internal/envtest/framework.go

```diff
@@ -0,0 +1,12 @@
+package envtest
```

Comment:
question: What was the motivation to move this single function to a new file? Seems to serve no purpose.

Reply:
I originally did have more logic here, but I ended up not needing it. I'd still rather split this out, and I was trying to find a filename that wasn't `utils`. I don't know, where do you think it belongs given it is shared between multi-cluster envtests, kind envtests, and all the rest?

Reply:
As far as I can say, it was in the envtest.go and was just fine there as this is still the same package. If it's a shared util class for many packages, it could be outside envtest pkg in that case. Like ``internal/shared`` or ``internal/env`` etc if you hate multiple utils classes.

Reply:
`internal/shared` also not a great name there Micke  😅 

It isn't shared between many packages, but again, it makes sense in the light of how I've refactored things for the multi-cluster and kind envtests. Can we please have that conversation there, as this can't rise to the level of an issue given it doesn't cause a functional problem.

Reply:
So rollback here and modify there? If it's not used outside the pkg, then there's even less of a reason to make a new file for it.

It could be useful in the tests though, if you make them in a test package and need to for some reason to access files in the RootDir(). For example, filepath.Join(RootDir(), "testfiles"). That would be outside envtest package.